plugin/trino-hive/src/test/java/io/trino/plugin/hive/TestDescView.java

```diff
@@ -0,0 +1,109 @@
+/*
```

Comment:
Please reuse an existing test class since starting a new HiveHadoop instance is slow.